dev-docs/bidders/taboola.md

```diff
@@ -27,14 +28,14 @@ The Taboola Adapter requires setup before beginning. Please contact us at prebid
 
 {: .table .table-bordered .table-striped }
 
-| Name          | Scope    | Description                                             | Example                    | Type         |
-|---------------|----------|---------------------------------------------------------|----------------------------|--------------|
-| `tagId`       | required | Tag ID / Placement Name <br>                            | `'Below The Article'`      | `String`     |
-| `publisherId` | required | Numeric Publisher ID <br>(as provided by Taboola)       | `'1234567'`                | `String`     |
-| `bcat`        | optional | List of blocked advertiser categories (IAB)             | `['IAB1-1']`               | `Array`      |
-| `badv`        | optional | Blocked Advertiser Domains                              | `'example.com'`            | `String Url` |
-| `bidfloor`    | optional | CPM bid floor                                           | `0.25`                     | `Float`      |
-| `endpointUrl` | optional | Endpoint Url (only if provided by Taboola)              | `https://example.com`      | `String`     |
+| Name              | Scope    | Description                                       | Example               | Type         |
+|-------------------|----------|---------------------------------------------------|-----------------------|--------------|
+| `tagId`           | required | Tag ID / Placement Name <br>                      | `'Below The Article'` | `String`     |
+| `publisherId`     | required | Numeric Publisher ID <br>(as provided by Taboola) | `'1234567'`           | `String`     |
+| `publisherDomain` | optional | Publisher Domain                                  | `'example.com'`       | `String`     |
+| `bcat`            | optional | List of blocked advertiser categories (IAB)       | `['IAB1-1']`          | `Array`      |
+| `badv`            | optional | Blocked Advertiser Domains                        | `'example.com'`       | `String Url` |
+| `bidfloor`        | optional | CPM bid floor                                     | `0.25`                | `Float`      |
```

Comment:
Looking at the code of your client-side adapter, I see that you're not supporting the floors module. Will open a PBJS issue for you to fix that. It's ok to accept floors on your parameters, but only as secondary to the conventions.

Reply:
will fix it in a separated PR